.yamllint.yaml

```diff
@@ -1,5 +1,8 @@
 extends: default
 
+ignore: |
+  /python/lsst/ctrl/bps/htcondor/etc/htcondor_defaults.yaml
```

Comment:
How come this file doesn't pass linting?

Reply:
It does. I just noticed similar entry in **ctrl_bps**' `.yamllint.yaml`for `bps_defaults.yaml` and assumed it may be needed here as well at some point. I mean I don't know/recall why `bps_defaults.yaml` was put on the ignore list in `.yamllint.yaml` by the file's creator Matthias Wittgen and my thinking was that at least in theory `htcondor_defaults.yaml` can also possibly contain some "offending" setting(s) in the future.

Reply:
We want it to be validated if at all possible so remove that line and see what happens.